src/RpcClient/Models/RpcInvokeResult.cs

```diff
@@ -43,6 +47,8 @@ public static RpcInvokeResult FromJson(JObject json)
             invokeScriptResult.Script = json["script"].AsString();
             invokeScriptResult.State = json["state"].TryGetEnum<VM.VMState>();
             invokeScriptResult.GasConsumed = json["gasconsumed"].AsString();
+            if (json.ContainsProperty("exception"))
+                invokeScriptResult.Exception = json["exception"].AsString();
```

Comment:
Here should be modified since if json contains `"exception":null,` **NullReferenceException** will come out like this:
```
{{"script":"DBQKo4e1Ppa3mJpjFDGgVt0fQKBC9hHADAliYWxhbmNlT2YMFM8xvN84WQYKa3qONu\u002BAYNfsnUslQWJ9W1I=","state":"HALT","gasconsumed":"3840690","exception":null,"stack":[{"type":"Integer","value":"2000000000000000"}],"tx":"00c30c1478b29a3a00000000000ea31200000000006f1c200001cd58aecfd335be1eb3d311e1dc800963dfd72d9f01003e0c140aa387b53e96b7989a631431a056dd1f40a042f611c00c0962616c616e63654f660c14cf31bcdf3859060a6b7a8e36ef8060d7ec9d4b2541627d5b5201420c4023e054a3d253bb0c9b5320922536f6dfc9fba3647d9f70012247abcb49d87b9082118f3417d06a8b198b341d1af0750f5b4bba451e6397a259cecf1cb02c1ac6290c2102fdf27a2eb8d9b069d61ae5a64dba13b89637c542d8c74c5d6bab493fe638d68f0b4195440d78"}}
```
![image](https://user-images.githubusercontent.com/38493437/96401840-0ef5e880-1207-11eb-918d-3821c58a6fbc.png)

Reply:
Fixed